python/vyos/kea.py

```diff
@@ -667,3 +673,25 @@ def kea_get_server_leases(config, inet, vrf_name, pools=[], state=[], origin=Non
                     data.pop(idx)
 
     return data
+
+def kea_build_client_class_test(config):
+    conditions = []
+
+    if "option82" in config:
+        if "circuit_id" in config["option82"]:
+            conditions.append("relay4[1].hex == 0x" + config["option82"]["circuit_id"].encode().hex().lower())
+        if "remote_id" in config["option82"]:
+            conditions.append("relay4[2].hex == 0x" + config["option82"]["remote_id"].encode().hex().lower())
+
+    is_first = True
+
+    test = ""
+    for condition in conditions:
+        if not is_first:
+            test += " and "
+
+        is_first = False
+
+        test += condition
```

Comment:
```suggestion

    test = " and ".join(conditions)
```

Am I missing anything here why a `.join()` would not suffice?